src/rules/require-sql-template.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+import assert from 'assert';
+import flora from 'flora-sql-parser';
+import get from 'lodash.get';
+
+/**
+ * Instances.
+ */
+
+const parser = new flora.Parser();
+
+/**
+ * Check if `literal` is an SQL query.
+ */
+
+function isSqlQuery(literal) {
+  if (!literal) {
```

Comment:
The problem here is that the library interprets empty strings as valid SQL. I'm doing some tests with a different (and smaller library) which does not need this workaround.